src/care/host_device_map.h

```diff
@@ -380,7 +386,7 @@ namespace care {
    {
       public:
          // default constructor
-         host_device_map() noexcept = default;         
+         host_device_map() noexcept {};         
```

Comment:
This is host only as well, so these implementations can be changed back to `= default`.